### PR TITLE
WV-2181 

### PR DESCRIPTION
**Pull request by jwhurley1**
## Description

Fixes:  Right click then Take a snapshot causes error.  

When right clicking on the map, coordinates are set in the Redux store.  When single-clicked off of the right-click menu and not selecting anything,  coordinates were still stored.  In `web/js/map/ui.js` there is a function `subscribeToStore` that runs `addMarkerAndUpdatedStore()` when the `TOGGLE_DIALOG_VISIBLE` is called.  This action is called when the snapshot button is clicked is it trying to close the little tooltip next to the location marker.  Since a location marker doesn't exist yet, it was throwing an error.  The new boolean `isMarkerPlaced` in `locationSearch` was added to show declare whether a marker exists on the map or not making the markers independent from coordinates existing in the store.

### Test 1
1. Pull down changes and run `npm start` and open [localhost:3000](http://localhost:3000).
2. Right click on map to bring up context menu, but don't select anything in the menu.
3. Click on "Take a Snapshot"
4. No error should show.  The Snapshot menu should open with the grayed out screen and a non-grayed out selection box.

### Test 2
1. Right click on map to bring up context menu, and place a marker on the map.
1. Right click again and do NOT select anything, leaving the menu open.
1. Click on "Take a Snapshot"
1. No error should show.  The Snapshot menu should open with the grayed out screen and a non-grayed out selection box.
1. The location marker's tooltip should minimize/disappear and the marker should be by itself.

### CODE REVIEW

File: web/js/containers/map-interactions/map-interactions.js [hunk 1]
```diff
@@ -30,13 +30,15 @@ class MapInteractions extends PureComponent {
...
-          <div id="wv-map" className={mapClasses} />
-        </ContextMenuTrigger>
         <RightClickMenu />
         <OlZoomButtons />
         <OlRotationButtons />
-        <OlCoordinates show={!isDistractionFreeModeActive} />
+        <ContextMenuTrigger id="context-menu-trigger">
+          <div id="wv-map" className={mapClasses} />
+        </ContextMenuTrigger>
+        {!isDistractionFreeModeActive && (
```
Comment (jasontk19) [inline web/js/containers/map-interactions/map-interactions.js:39]: I think there may have been a bad conflict resolution here, it looks like you undid my changes that should read as follows: 
```
<OlCoordinates show={!isDistractionFreeModeActive} />
```

**Review by jasontk19 — CHANGES_REQUESTED** [review]
Looks like the bug is fixed but I think there were some unintentional changes made during the rebase.

File: web/js/containers/map-interactions/map-interactions.js (hunk 1 above)
Comment (jasontk19) [reply web/js/containers/map-interactions/map-interactions.js:39]: Fixing this should fix the broken e2e tests as well.  You can confirm locally by doing:

1. Go to info menu
2. Enter distraction free mode
3. Click the eyeball icon top right to exit distraction free mode

Presently, this results in an error on this branch.  Should be fixed with the above change addressed.

**Review by minniewong — CHANGES_REQUESTED** [review]
The fix works really well but like Jason noted I am seeing some other unexpected issue. The map coordinates box that's in the lower right corner is no longer there.

File: web/js/containers/map-interactions/map-interactions.js (hunk 1 above)
Comment (jwhurley1) [reply web/js/containers/map-interactions/map-interactions.js]: This worked.  Good eye catching this.  I was manually accepting a lot of rebase conflicts and they were all in favor of my local changes vs remote changes.  I ended up accepting all the local changes blindly...and this is what I get lol.

**Comment by jwhurley1** [comment]
@minniewong 
> The fix works really well but like Jason noted I am seeing some other unexpected issue. The map coordinates box that's in the lower right corner is no longer there.

After making the change Jason K. mentioned, I see the map coordinates box.  (Hopefully this is the one you're talking about)
image

**Review by minniewong — APPROVED** [review]
yep! it's back and looks good!

**Comment by jwhurley1** [comment]
I just found another bug, which I'm working on now.

1. Right click and "Add Marker"
2. Right click again and click off of the context menu.
3. Marker disappears.

**Pull request closed without merging**